### ISSUE TITLE
Infinite loop error when adding "Cache-Control" HTTP Header

### ISSUE THREAD

**Issue by flsilva** [tonyhb/tectonic#88]
I'm working on a simple app where I have a Projects Model:

```
import { Model } from 'tectonic'

export default class Project extends Model {
  static modelName = 'project'

  static fields = {
    id: 0,
    name: ''
  }
}
```

Source definition:

```
manager.drivers.fromSuperagent([
  {
    meta: {
      url: apiPath + '/projects',
      method: 'GET',
    },
    returns: ProjectModel.list(),
  }
])
```

Query data:

```
@load((props) => ({
  projects: ProjectModel.getList()
}))
```

Everything works fine without cache headers (every time I visit that route a new request is successfully done). Once I add `Cache-Control:max-age=7200, public` to the response, I get an infinite loop error from React:

```
warning.js:60 Uncaught RangeError: Maximum call stack size exceeded
    at warning (warning.js:60)
    at callHook (ReactDebugTool.js:29)
    at emitEvent (ReactDebugTool.js:39)
    at Object.onBeforeUpdateComponent (ReactDebugTool.js:326)
    at Object.receiveComponent (ReactReconciler.js:115)
    at Object.updateChildren (ReactChildReconciler.js:109)
    at ReactDOMComponent._reconcilerUpdateChildren (ReactMultiChild.js:208)
    at ReactDOMComponent._updateChildren (ReactMultiChild.js:312)
    at ReactDOMComponent.updateChildren (ReactMultiChild.js:299)
    at ReactDOMComponent._updateDOMChildren (ReactDOMComponent.js:936)
```
﻿
Debugging it I can see a `@@tectonic/update-query-statuses` action being dispatched infinitely, with following `payload` value:
​
```
Query(Model: project, Fields: *, Params: {}, Body: undefined, QueryType: GET), ReturnType: LIST): Object
  status: "SUCCESS"
```

Let me know if you need more info.

I hope this is a quick fix, I just started playing with the lib and it looks awesome, just what I was looking for, congrats for the great work.

**Comment by tonyhb** [comment]
Ty, taking a look! :)

**Comment by tonyhb** [comment]
Fixed in #103
